tests/subsys/tracing/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+
+if(BOARD MATCHES "qemu_.*")
```

Comment:
where is this output being used?

Reply:
not used yet, I will remove this.  thanks .

Reply:
Have fixed this, thanks.